x/payment/client/testutil/integration_test.go

```diff
@@ -131,7 +132,7 @@ func (s *IntegrationTestSuite) TestSubmitWirePayForMessage() {
 
 				events := txResp.Logs[0].GetEvents()
 				for i := 0; i < len(events); i++ {
-					s.Equal("/payment.MsgPayForMessage", events[i].GetAttributes()[0].Value)
+					s.Equal(types.URLMsgPayforMessage, events[i].GetAttributes()[0].Value)
```

Comment:
this change is unrelated to this PR, but we should definitely be using a constant here.